meeting_notes/2024/2024_10_01.md

```diff
@@ -0,0 +1,75 @@
+# Agenda
+
+## Reminder: Hackathon on Friday
+
+Goal: 'dressed rehearsal' of the Surface API to see if it's ready from prime time.
+
+Rémi: I will be "on" for most of the day, so that people can drop by at any time and I will try to prepare 'tracks' for 'things to do/review if I have 30 minutes / 1 hour / 3 hours'.
+
+- Advertise hackathon: twitter and others...
+
+## News:
+
+### hacktober fest
+
+Rémi: we could try to add Nilearn to the pool of repos that participate to the next hacktoberfest. From our side it requires adding a tag to the repo and some extra labels to the issues / PR that are involved in the hacktoberfest. I have a few issues in mind whose wording I could improve to make them more explicit on what is required.
+
+More info here: https://hacktoberfest.com/participation/#maintainers
+
+**ACTION**: let's try it and see if we are flooded with SPAM
+
+### 2 nilearn google calendar to track public and private events
+
+Rémi: You should have received emails to give you "write" access to both. Let me know if you did not receive anything or if I should add another email address for you. @Elizabeth I also added the nilearn.events@gmail.com as one of the admin (I think you are in control of it, right?).
+ 
+  - public: https://calendar.google.com/calendar/embed?src=87f409f6c8543a93a45d4c136e011abe6bcf78adfc1bbf665fc3bd2f8bb0f5ba%40group.calendar.google.com&ctz=Europe%2FParis
+
+  - private: https://calendar.google.com/calendar/embed?src=945ae93030a36f912a00a67b1d752141a3dc50cc05b6de46bc2af3ab66a081d5%40group.calendar.google.com&ctz=Europe%2FParis
+
+### Youtube channel
+
+Rémi: I am not sure but this looks like the "official" channel, no?
+
+https://www.youtube.com/@nilearnevents5116/videos
+
+Is this linked to the nilearn.events@gmail.com address ? @Elisabeth do you know?
```

Comment:
```suggestion
Is this linked to the nilearn.events@gmail.com address ? @Elizabeth do you know?
```

Reply:
Maybe we should update the minutes with what was discussed ? Or did you want these as just the agendas ?